backend/src/schema/resolvers/embeds.js

```diff
@@ -11,16 +11,17 @@ export default {
     ...undefinedToNullResolver([
       'type',
       'title',
+      'description',
       'author',
       'publisher',
-      'date',
-      'description',
       'url',
+      'date',
       'image',
       'audio',
       'video',
       'lang',
-      'html',
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-17T16:15:23Z" title="Tuesday, September 17th 2019, 6:15:23 pm +02:00">Sep 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -11,16 +11,17 @@ export default {
     ...undefinedToNullResolver([
       'type',
       'title',
+      'description',
       'author',
       'publisher',
-      'date',
-      'description',
       'url',
+      'date',
       'image',
       'audio',
       'video',
       'lang',
-      'html',
```


Hm, I think for the trusted oembed providers in `providers.json` we can return plain html to the client.